notebooks/stable-video-diffusion/stable-video-diffusion.ipynb

```diff
@@ -9,7 +9,7 @@
     "# Image to Video Generation with Stable Video Diffusion\n",
```

Comment:
<p>could you please check title level for added section. Jugging by Table of content, it looks like you need to Remove one # from Quantization title (for logical separation from Run Video Generation section)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2041/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>